backend/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.1'
+
+services:
+
+  database:
+    image: postgres:15.3-bullseye
+    restart: always
```

Comment:
You want to also expose its postgres port so one can connect to this dev db using another tool. Very convenient